src/TLuaInterpreter.cpp

```diff
@@ -5182,6 +5182,23 @@ int TLuaInterpreter::getMudletHomeDir( lua_State * L )
     return 1;
 }
 
+// returns search paths for LuaGlobal itself to look at when loading other modules
+// follows the principle of closest paths to the binary first, furthest away lasy
+int TLuaInterpreter::getMudletLuaDefaultPaths( lua_State * L )
+{
+    lua_newtable( L );
+    lua_createtable(L,2,0);
+    // add filepath relative to the binary itself (one usecase is AppImage on Linux)
+    QString nativePath = QDir::toNativeSeparators( QCoreApplication::applicationDirPath() + "/mudlet-lua/lua/" );
```

Comment:
I have no idea about C++, but this is not the correct path to look at for macOS. The code a few lines below works for locating the LuaGlobal.lua, though:

```.cpp
 #if defined(Q_OS_MAC)
     QString path = QCoreApplication::applicationDirPath() + "/../Resources/mudlet-lua/lua/LuaGlobal.lua";		     QString path = QCoreApplication::applicationDirPath() + "/../Resources/mudlet-lua/lua/LuaGlobal.lua";
 #else
```

Could you add this in? If so, we _might_ be able to get rid of the run_mudlet helper.